### ISSUE TITLE
WebTorrent Goes Grey

### ISSUE THREAD

**Issue by AmbrociousXP** [feross/webtorrent-desktop#548]
**What version of WebTorrent Desktop?** 0.5.1 (0.93.4)

**What operating system and version?** Windows 7 64bit

**What did you do?** Turned on Webtorrent

**What did you expect to happen?** To see the torrents continue downloading.

**What actually happened?** The area where the torrents are USUALLY at displayed the torrents briefly but then it all went grey, showing nothing on the screen (the program) not even the normal place where you can paste links was there.

![screenshot_2](https://cloud.githubusercontent.com/assets/1117168/15440835/d4188b74-1e94-11e6-8cf6-6ec1036ceeee.png)

**Comment by feross** [comment]
Can you click View > Developer > Developer Tools, then click on the Console
tab and copy out the text and paste it here?

That would really help us to debug this!

**Comment by AmbrociousXP** [comment]
![screenshot_3](https://cloud.githubusercontent.com/assets/1117168/15441206/d54b9c5a-1e96-11e6-8c2a-61ffbc785dc1.png)


This was it but nothing seems to be in there.

**Comment by AmbrociousXP** [comment]
OK so I restarted Weborrent and opened up the console as you said BEFORE the torrents disappeared off the list. This is what I saw:

![screenshot_1](https://cloud.githubusercontent.com/assets/1117168/15441642/46ffee26-1e99-11e6-8caa-114d7bc464c3.png)

**Comment by AmbrociousXP** [comment]
go Object
index.js:451 Command line args: Array[0]
index.js:451 onWindowShow
index.js:488 loaded state from C:\Users\YAKKITYY YAK\AppData\Local\WebTorrent\config.json
index.js:719 start torrenting 1 C:\Users\YAKKITYY YAK\AppData\Local\WebTorrent\Torrents\1dd24b435d2e53304e676dc62a9d34addddffb1e.torrent
index.js:102 init: 1466.000ms
index.js:451 webtorrent: queueing wt-start-torrenting
index.js:451 Checking for update
index.js:451 setProgress 0
index.js:451 setBadge 
index.js:451 Update not available
index.js:451 sending 1 queued messages from the main win to the webtorrent window
index.js:451 webtorrent: sent queued wt-start-torrenting
index.js:785 got infohash for existing torrent 1
index.js:451 webtorrent: got wt-infohash
index.js:451 webtorrent: got wt-metadata
index.js:451 webtorrent: got wt-progress
index.js:451 webtorrent: sent wt-generate-torrent-poster
index.js:1235 warning: getaddrinfo ENOTFOUND tracker.thepiratebay.org tracker.thepiratebay.org:80
index.js:451 webtorrent: got wt-warning
index.js:1235 warning: getaddrinfo ENOTFOUND tracker.publicbt.com (udp://tracker.publicbt.com:80)
index.js:451 webtorrent: got wt-warning
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.02918068257814292
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.047610587364338444
index.js:1235 warning: getaddrinfo ENOTFOUND tracker-ccc.de (udp://tracker-ccc.de:6969)
index.js:451 webtorrent: got wt-warning
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.07064796834708285
index.js:451 toggleDevTools
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.09214952393097764
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.12286603190797019
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.15972584148036123
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.19504982565390266
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.22269468283319596
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.24419623841709073
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.2764485717929329
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.3040934289722262
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.337881587746918
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.37474139731930906
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.4116012068917001
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.44846101646409114
index.js:1235 warning: tracker request timed out (started) (udp://tracker.openbittorrent.com:80)
index.js:451 webtorrent: got wt-warning
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.4837850006376326
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.5052865562215274
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.53600306419852
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.5759345245686103
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.6112585087421516
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.6481183183145427
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.6834423024880841
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.712622985066227
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.7464111438409189
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.7878784296098588
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.8262740645810994
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.8631338741534905
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.9061369853212801
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.9383893186971223
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress 0.9752491282695133
index.js:451 webtorrent: got wt-ready
index.js:451 webtorrent: got wt-ready-1dd24b435d2e53304e676dc62a9d34addddffb1e
index.js:451 webtorrent: got wt-progress
index.js:451 setProgress -1
index.js:451 webtorrent: got wt-done
index.js:451 setBadge 1
index.js:1235 warning: getaddrinfo ENOTFOUND tracker.thepiratebay.org tracker.thepiratebay.org:80
index.js:451 webtorrent: got wt-warning
index.js:1235 warning: getaddrinfo ENOTFOUND tracker.publicbt.com (udp://tracker.publicbt.com:80)
index.js:451 webtorrent: got wt-warning
index.js:451 webtorrent: got wt-poster
index.js:451 webtorrent: got wt-file-modtimes
index.js:451 webtorrent: got wt-progress

**Comment by AmbrociousXP** [comment]
Upon tinkering a bit with it I discovered that AFTER one of my torrents finished, that is when the program displays nothing but a grey screen, unable to paste any further links.

**Comment by drz01** [comment]
I can confirm the same with the Mac version. Rebooting and re-installing did not resolve issue.

**Comment by feross** [comment]
@drz01 Can you open Terminal and run the following command:

```
pbcopy < ~/Library/Application\ Support/WebTorrent/config.json
```

This will copy the WebTorrent configuration file to your clipboard. Then send me an email (address is on my GitHub profile page) and paste the config file. (I ask for an email, since this file can be pretty large, too big to post here).

**Comment by feross** [comment]
The errors that you were experiencing are due to a race condition in how the app is loaded. In order to make app startup faster, I made the main window report itself as being ready a bit sooner than I should have.

This is fixed in this PR: https://github.com/feross/webtorrent-desktop/pull/613 which will be released in WebTorrent Desktop v0.7.0, **TOMORROW**.

If your WebTorrent app is broken (gray screen on startup) -- then you can try starting it up and leaving it open for around 5 minutes, then restart it. This should give the auto-updater enough time to update to WebTorrent Desktop v0.7.0. But you'll have to wait for v0.7.0 to be released tomorrow.

Alternatively, you can delete your WebTorrent config file to get the app working again immediately. Just quit the app, delete the following folder, then restart the app:

OS X: `~/Library/Application Support/WebTorrent`
Linux: `~/.config/WebTorrent`
Windows: `C:\Users\YOUR_USER\App Data\Local\WebTorrent`
